src/lib/y2users/linux/writer.rb

```diff
@@ -124,15 +130,38 @@ def write
       CHPASSWD = "/usr/sbin/chpasswd".freeze
       private_constant :CHPASSWD
 
+      # Whether given user does not exist yet
+      #
+      # FIXME: choose a better criteria to identify a new user. An internal Y2User::User id?
+      #
+      # @param user [Y2User::User]
+      # @return [Boolean] true if there is a user with same name in @initial_config; false otherwise
+      def new_user?(user)
+        @initial_users_names ||= initial_config.users.map(&:name)
+
+        !@initial_users_names.include?(user.name)
+      end
+
+      # Executes the command for creating the user
+      #
+      # @param user [Y2User::User]
       def add_user(user)
+        return unless new_user?(user)
+
         Yast::Execute.on_target!(USERADD, *useradd_options(user))
+      rescue Cheetah::ExecutionFailed => e
+        log.error("Error creating user '#{user.name}' - #{e.message}")
```

Comment:
is log enough? what about using at least that Issues List @imobachgs adds to yast2?

Reply:
I was discussing it with @ancorgs last week: looks like a previous job for the validator.

Reply:
Well, my POV is that if it should not happen as validator already catch it, then raise exception as we have bug in validator....and if it can happen, we need to report it. Silently ignore it is the worst case as it means that user do not know about failure and result is not what user expect.

Reply:
Yes, you're right. What I meant was not to perform validations _again_, but it's true that we should provide some kind of information to the user at some point :) But I was thinking to do it later, in another step/PR

Reply:
Creating and returning the list of found issues. See https://github.com/yast/yast-users/pull/255/commits/084fe46c45066171e1870c5efd978fa8a52e237f

It's simple / dummy by now on purpose. Let's see how evolve it while the writer grow.